examples/local/cert-operator-lab-chart/templates/deployment.yaml

```diff
@@ -24,7 +24,8 @@ spec:
           containerPort: 8000
         args:
         - daemon
-        - --service.vault.config.address=http://${VAULT_HOST}:8200
-        - --service.vault.config.token=${VAULT_TOKEN}
+        - --service.vault.config.address=http://vault:8200
```

Comment:
Is the vault host fixed now? Or was this leftover config before?

Reply:
`vault` should be used for local setup, anyway i've added a `vaultHost` entry in values so that it can be changed if needed.